mmcv/runner/runner.py

```diff
@@ -254,6 +255,7 @@ def train(self, data_loader, **kwargs):
         self.model.train()
         self.mode = 'train'
         self.data_loader = data_loader
+        self.active_loader = data_loader
```

Comment:
What is the difference from `self.data_loader` and `self.active_loader`? It may not be necessary to assign a new attribute here.

Reply:
Sorry, didn't see that. My bad.